web_widget_child_selector/tests/test_widget_child_selector.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestWidgetChildSelector(TransactionCase):
+
+    def setUp(self):
+        super().setUp()
+        self.partner_1 = self.env['res.partner'].create({'name': 'P1'})
```

Comment:
I know you have only one test so I won't push to use SavepointCase but please, disable tracking as you are creating many record :pray: